coffee-jpa/src/main/java/hu/icellmobilsoft/coffee/jpa/annotation/Transactional.java

```diff
@@ -0,0 +1,23 @@
+package hu.icellmobilsoft.coffee.jpa.annotation;
+
+import jakarta.enterprise.inject.Stereotype;
+import jakarta.interceptor.InterceptorBinding;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Customized {@link jakarta.transaction.Transactional} annotation with rollbackOn at {@link Throwable}.
+ *
+ * @author speter555
+ * @since 2.0.0
```

Comment:
ezt elso korben inkabba v1-r kellene tolni majd a v2-re. Verziok is egyenlore ugy fogjak kovetni egymast

Reply:
javítottam